components/console/events.rst

```diff
@@ -20,6 +20,11 @@ the wheel, it uses the Symfony EventDispatcher component to do the work::
     $application->setDispatcher($dispatcher);
     $application->run();
 
+.. caution::
+
+    Console application lifecycle events are only triggered by the main command being executed. Inner or nested
```

Comment:
I think we should call them "Console events" (just like we did in the versionadded directive).